pkg/proxy/proxy.go

```diff
@@ -496,43 +505,67 @@ func (r *RuntimeProxy) containerStats(ctx context.Context, method string, req, r
 	return resp, nil
 }
 
-func (r *RuntimeProxy) handleImage(ctx context.Context, method string, req, resp CRIObject) (interface{}, error) {
+// We don't want to force the user to prefix image names so instead, prefer
+// to say the image is not present if it's not available to all CRIs
+func (r *RuntimeProxy) handleImageStatus(ctx context.Context, method string, req, resp CRIObject) (interface{}, error) {
 	in := req.(ImageObject)
-	client, unprefixed, err := r.clientForImage(in.Image(), true)
-	if client == nil {
-		// the client is offline
-		return resp, nil
+	for i := range r.clients {
+		client, err := r.clientAtIndex(i)
+		if err != nil {
+			continue
+		}
+		imageName := in.Image()
+		_, err = client.invokeWithErrorHandling(ctx, method, req, resp)
+		if err != nil {
+			glog.Errorf("Error in ImageStatus for client %s: %v", client.getID(), err)
+			return nil, err
+		}
+		if out, ok := resp.(ImageStatusResponse); ok && out.Image() != nil {
+			img := out.Image().(Image)
+			if len(img.RepoDigests()) > 0 {
+				imageName = img.RepoDigests()[0]
+			}
+			r.setImageNameById(img.Id(), imageName, true)
+			out.SetImage(out.Image())
```

Comment:
Does this do anything? At first glance, it just seems to re-set Image to be the same thing.

Reply:
Right, yep, I was copying what was done before but before they set the image by adding a prefix, I dropped the prefix so... we set the image to the image.